docs/embeddable-uis/element-login.mdx

```diff
@@ -629,6 +629,64 @@ authentication solution.
 permit.elements.logout();
 ```
 
+---
+### Support for Private Browsing Mode
+
+Permit Element is now compatible with private browsers, such as Chrome Incognito Mode and Safari Private Browsing.
+
```

Comment:
To do that you need to change a few things from the regular cookie-based permit elements